src/Snowcap/Emarsys/Client.php

```diff
@@ -418,6 +418,21 @@ public function getContactsFromContactList($listId, array $data)
     }
 
     /**
+     * Checks whether a specific contact is included in the defined contact list.
+     *
+     * @param integer $contactId
+     * @param integer $listId
```

Comment:
Please change `integer` into `int` in two lines above in order to march coding style of a project.